camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -938,6 +939,35 @@ def pt2line(x0, y0, x1, y1, x2, y2):
         else:
             return None
 
+    def update_mask(self, img, screen, radius = 35):
+        """
+        :param img: source image
+        :type img: :class:`cvMat`
+
+        :param screen: display
+        :type screen: :class:`cvtColor`
+
+        :param radius: radius of the green circle which shows where corner detection has occurred
+        :type radius: :class:`int`
+
+        Update or if None create a red mask which shows where no corner detection has occurred
```

Comment:
in the reST  doc string format the function description is before the arguments. Return is missing, but its None anyways